specification/app/resource-manager/Microsoft.App/ContainerApps/stable/2026-01-01/AgentSpacesConnectors.json

```
@@ -0,0 +1,410 @@
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List secrets for an agent space connector": {
+            "$ref": "./examples/AgentSpacesConnectors_ListSecrets.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/agentSpaces/{agentSpaceName}/listConnectorsWithSecrets": {
+      "post": {
+        "tags": [
+          "AgentSpacesConnectors"
+        ],
+        "summary": "List all secrets for AgentSpace Connectors.",
+        "operationId": "AgentSpacesConnectors_ListAllSecrets",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "agentSpaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[A-Za-z]([-A-Za-z0-9]*[A-Za-z0-9])$"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AgentSpaceDataConnectorCollection"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List all secrets for agent space connectors": {
+            "$ref": "./examples/AgentSpacesConnectors_ListConnectorsWithSecrets.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AgentSpaceDataConnectorCollection": {
+      "description": "Collection of AgentSpaceDataConnector",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of AgentSpaceDataConnector",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AgentSpaceDataConnector"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "AgentSpaceDataConnector": {
```

Needs to be named as 
```suggestion
    "AgentSpaceConnector": {
```